gulpfile.ts

```diff
@@ -2,43 +2,44 @@ import * as gulp from 'gulp';
 import * as mocha from 'gulp-mocha';
 import * as plumber from 'gulp-plumber';
 import * as ts from 'gulp-typescript';
-import tslint from 'gulp-tslint';
+import * as eslint from 'gulp-eslint';
 import * as del from 'del';
 
 const tsProject = ts.createProject('tsconfig.json');
 
 gulp.task('ts', () => {
-  return gulp.src(['src/**/*.ts', '!src/**/templates/**/*.ts']).pipe(tsProject()).js.pipe(gulp.dest('generators'));
+  return gulp.src([ 'src/**/*.ts', '!src/**/templates/**/*.ts' ]).pipe(tsProject()).js.pipe(gulp.dest('generators'));
 });
 
 gulp.task('templates', () => {
   return gulp.src(['src/package/templates/**/*', 'src/plugin/templates/**/*'], { base: 'src' }).pipe(gulp.dest('generators'));
 });
 
-gulp.task('tslint', function () {
-  return gulp.src(['src/**/*.ts', '!generators/**/templates/**/*.ts'])
-    .pipe(tslint({ formatter: 'prose' }))
-    .pipe(tslint.report());
+gulp.task('eslint', () => {
+  return gulp.src([ 'src/**/*.ts', '!generators/**/templates/**/*.ts' ])
+    .pipe(eslint())
```

Comment:
How are we able to call eslint if it's imported using `* as`?

Reply:
It's a commonjs interopt thing. I'll see if I can dig up the docs, but we use it a lot in bedrock server as well.